ginkgo.Jenkinsfile

```diff
@@ -52,6 +55,7 @@ pipeline {
                     junit 'test/*.xml'
                     sh 'cd test/; vagrant destroy -f'
                     sh 'cd test/; K8S_VERSION=1.6 vagrant destroy -f'
+                    sh 'cd test/; K8S_VERSION=1.8 vagrant destroy -f'
```

Comment:
Shouldn't be a line with `1.7`?

Reply:
Hello. 

1.7 is the default env variable, so is deleted on the line 52